indra/newview/llfavoritesbar.cpp

```diff
@@ -285,22 +286,42 @@ class LLFavoriteLandmarkMenuItem : public LLMenuItemCallGL
 class LLFavoriteLandmarkToggleableMenu : public LLToggleableMenu
 {
 public:
-	virtual BOOL handleDragAndDrop(S32 x, S32 y, MASK mask, BOOL drop,
-								   EDragAndDropType cargo_type,
-								   void* cargo_data,
-								   EAcceptance* accept,
-								   std::string& tooltip_msg)
-	{
-		*accept = ACCEPT_NO;
-		return TRUE;
-	}
+    // virtual
+    BOOL handleDragAndDrop(S32 x, S32 y, MASK mask, BOOL drop, EDragAndDropType cargo_type,
+        void* cargo_data, EAcceptance* accept, std::string& tooltip_msg) override
+    {
+        LLToggleableMenu::handleDragAndDrop(x, y, mask, drop, cargo_type, cargo_data, accept, tooltip_msg);
+        return TRUE;
+    }
+
+    // virtual
+    BOOL handleHover(S32 x, S32 y, MASK mask) override
+    {
+        mIsHovering = true;
+        LLToggleableMenu::handleHover(x, y, mask);
+        mIsHovering = false;
+        return TRUE;
+    }
+
+    // virtual
+    void setVisible(BOOL visible) override
```

Comment:
feels a bit unsafe to me, might be a good idea to add LLToggleableMenu::setVisible(FALSE) in destructor